Client/src/services/saveLastCityToLS.js

```diff
@@ -0,0 +1,26 @@
+const saveLastCityToLS = (city, id) => {
+  let stuffInLs = JSON.parse(localStorage.getItem('lastCity'));
+
+  function checkIfCityExists() {
+    let exists = Boolean;
```

Comment:
Boolean - is a constructor.  you basically assign a function to your boolean value. use true or false.